### PR TITLE
Add assignment 1 by Everly (Arrays)

### PR DESCRIPTION
**Pull request by everly-gif**
Completed all assignment questions.
Arrays - **all 22 questions** ✔
Sample test cases -**tested and passed** ✔
By **Everly Precia Suresh**

### CODE REVIEW

File: Everly Precia Suresh/Arrays/delete_duplicates.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+int main(){
+    int n;
+    cout<<"Enter number of elements : ";
+    cin>>n;
+    int ar[n],freq[n];
+    cout<<"Enter Elements: ";
+
+    for(int i=0;i<n;i++){ //read
+        cin>>ar[i];
+        freq[i]=-1;
```
Comment (Vishal-raj-1) [inline Everly Precia Suresh/Arrays/delete_duplicates.cpp:12]: What's the use of freq array?

Comment (everly-gif) [reply Everly Precia Suresh/Arrays/delete_duplicates.cpp:12]: the freq array has count of each elements present in inputted array. for eg if ar[]={1,2,1,1,2} then freq array will contain = {3,2,0,0,0} (i.e 1=3 count , 2=2 count). The elements that contain zeros indicates that the element is already counted before.

**Review by Vishal-raj-1 — APPROVED** [review]
Awesome Man !!

File: Everly Precia Suresh/Arrays/delete_duplicates.cpp (hunk 1 above)
Comment (everly-gif) [reply Everly Precia Suresh/Arrays/delete_duplicates.cpp]: changed the code ! thanks for pointing this out

**Pull request merged**